src/pages/homeCollection/assignKits.js

```diff
@@ -85,7 +85,14 @@ const assignKitsTemplate = async (name) => {
   scannedBarcode2.onpaste = e => e.preventDefault();
   scannedBarcode2.addEventListener("input", (e) => {
     const scannedBarcodeValue = document.getElementById('scannedBarcode').value.trim();
-    const scannedBarcode2Value = e.target.value.trim();
+    let scannedBarcode2Value = e.target.value.trim();
+    
+    // Trim FedEx tracking numbers (1033)
+  // Already done for the scannedBarcode input in checkTrackingNumberSource
+    if (scannedBarcode2Value.length === 34) {
+      scannedBarcode2Value = scannedBarcode2Value.slice(-12);
+      e.target.value = scannedBarcode2Value;
```

Comment:
nit - these two lines could be combined to one

Reply:
I use the scannedBarcode2Value later on in the code for cross-reference, so I wanted to set its value and also set e.target.value to update the input itself.

Reply:
Didn't see that, thanks for the info